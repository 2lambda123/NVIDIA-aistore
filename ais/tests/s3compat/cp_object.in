echo "0123456789" > $OBJECT.txt
aws --endpoint-url http://localhost:8080/s3 s3 mb s3://$BUCKET_1
aws --endpoint-url http://localhost:8080/s3 s3 mb s3://$BUCKET_2
aws --endpoint-url http://localhost:8080/s3 s3 cp $OBJECT.txt s3://$BUCKET_1/$OBJECT // IGNORE
aws --endpoint-url http://localhost:8080/s3 s3 cp s3://$BUCKET_1/$OBJECT s3://$BUCKET_1/$OBJECT_1 // IGNORE
aws --endpoint-url http://localhost:8080/s3 s3 cp s3://$BUCKET_1/$OBJECT s3://$BUCKET_2/$OBJECT_3 // IGNORE
aws --endpoint-url http://localhost:8080/s3 s3 ls s3://$BUCKET_1
aws --endpoint-url http://localhost:8080/s3 s3 ls s3://$BUCKET_2
aws --endpoint-url http://localhost:8080/s3 s3 rb s3://$BUCKET_1
aws --endpoint-url http://localhost:8080/s3 s3 rb s3://$BUCKET_2
rm $OBJECT.txt // IGNORE
