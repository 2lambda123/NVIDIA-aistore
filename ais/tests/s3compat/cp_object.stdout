make_bucket: $BUCKET_1
make_bucket: $BUCKET_2
^.*11.*$
^.*11.*$
^.*11.*$
remove_bucket: $BUCKET_1
remove_bucket: $BUCKET_2
